issue.go

```diff
@@ -1332,6 +1337,8 @@ func InitIssueWithMetaAndFields(metaProject *MetaProject, metaIssuetype *MetaIss
 				issueFields.Unknowns[jiraKey] = []Component{{Name: value}}
 			case "option":
 				issueFields.Unknowns[jiraKey] = []map[string]string{{"value": value}}
+			case "id":
+				issueFields.Unknowns[jiraKey] = []map[string]string{{"id": value}}
```

Comment:
I'm wondering, can't we write this as
```go
issueFields.Unknowns[jiraKey] = []Id{{Id: value}}
```
That would match the approach on line 1337 with `[]Component{{Name: value}}`.
We can arguably also change line 1339 to `[]Option{{Value: value}}`.